constants/mobile_constants.asm

```diff
@@ -61,3 +61,10 @@ DEF MOBILE_PHONE_NUMBER_LENGTH EQU 20
 ; Maximum amount of time allowed for mobile battles each day
 DEF MOBILE_BATTLE_ALLOWED_SECONDS EQU 0
 DEF MOBILE_BATTLE_ALLOWED_MINUTES EQU 10
+
+; Trade corner request size
+; DION addr $1e + request $8 + Name $5
+; + party struct $30 + OT $5 + NICK $5
+; + JP Mail struct $2a
```

Comment:
Should these be constants?

Reply:
They go along with the struct mentioned above... So probably.